src/assets/scss/overrides.scss

```diff
@@ -135,8 +135,17 @@
   border: thin solid $app-red !important;
 }
 
+// blue button: blue on white
+.button-text {
+  color: $app-blue !important;
```

Comment:
Can you use `@extend` to keep the same styles as button-blue and just add what you need?

Example: https://github.com/bcgov/namerequest/blob/e6a86c444d2bc113db98c3b649585485164dd965/src/assets/scss/base.scss#L117

Reply:
This code base just keeps getting better and better! This morning I felt :( about this app, but now I feel :)